src/metax_api/migrations/0030_auto_20201028_1337.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 3.1.2 on 2020-10-28 10:32
+
+from django.db import migrations, connection
+import csv
+import json
+import logging
+from django.utils import timezone
+
+
+logger = logging.getLogger(__name__)
+
+def remove(apps, schema_editor):
+    CatalogRecord = apps.get_model('metax_api', 'CatalogRecord')
+
+    syke_objs = CatalogRecord.objects \
+        .filter(data_catalog__catalog_json__identifier='urn:nbn:fi:att:data-catalog-harvest-syke',
+        research_dataset__other_identifier__0__notation='') \
+        .exclude(research_dataset__other_identifier__0__old_notation='')
+
+    for cr in syke_objs:
+        try:
+            cr.user_modified = "migration_0029"
+            delete_d(cr)
```

Comment:
user modified might be misleading if this is migr num 30

Reply:
fixed filename